docs/getting-started-guides/ubuntu/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Kubernetes on Ubuntu
+redirect_from: /docs/getting-started-guides/ubuntu-calico/
```

Comment:
Sorry, we just changed to using a server-side redirect mechanism, so you can put the redirect in the `/_redirects` file. Just add a line:

```
/docs/getting-started-guides/ubuntu-calico    /docs/getting-started-guides/ubuntu    301
```